### ISSUE TITLE
Context prediction pre-training loss weighting

### ISSUE THREAD

**Issue by agamemnonc** [snap-stanford/pretrain-gnns#24]
This is a nit-pick and doesn't really matter when using default arguments (`neg_samples=1`), but shouldn't the combined loss here:
https://github.com/snap-stanford/pretrain-gnns/blob/7bb81b5cc2d37241ee72cbfa40fbd89b0cc2394f/chem/pretrain_contextpred.py#L93

instead be normalized as follows?

`loss = loss_pos + (1/args.neg_samples) * loss_neg`

**Comment by agamemnonc** [comment]
Please ignore the issue, I have now worked it out.
